src/views/EdgeApplicationsCacheSettings/FormFields/FormFieldsEdgeApplicationCacheSettings.vue

```diff
@@ -481,12 +481,12 @@
             <RadioButton
               v-model="cacheByQueryString"
               :disabled="disabledQueryStringOptions(queryStringOption)"
-              :inputId="queryStringOption.value"
+              :inputId="`cbqs-${queryStringOption.value}`"
```

Comment:
I know it may be out of scope of this PR, but ...
Is it possible to use full/extended name? `cbqs` doesn't add meaning to the code. Makes it harder to read and perform maintenance.

Reply:
No problemo!